python/delta/integration_tests/unity-catalog-commit-coordinator-integration-tests.py

```diff
@@ -54,54 +63,195 @@
+            f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}", True)
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(5, )]))
+
+    def test_insert_into_overwrite_replace_where(self):
+        single_col_df = spark.createDataFrame([(5, )], schema=["id"])
+        single_col_df.write.mode("overwrite").option("replaceWhere", "id > 1").insertInto(
+            f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}", True)
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(1, ), (5, )]))
+
+    def test_insert_into_overwrite_partition_overwrite(self):
+        single_col_df = spark.createDataFrame([(5,)], schema=["id"])
+        single_col_df.write.mode("overwrite").option(
+            "partitionOverwriteMode", "dynamic").insertInto(
+            f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}", True)
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(5,)]))
+
+    def test_save_as_table_append_existing_table(self):
+        single_col_df = spark.createDataFrame(
+            [(4, ), (5, )], schema=StructType([StructField("id", IntegerType(), True)]))
+        single_col_df.write.format("delta").mode("append").saveAsTable(
+            f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}")
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(1, ), (2, ), (3, ), (4, ), (5, )]))
+
+
+    # Setting mode to append should work, however cc tables do not allow path based access.
+    def test_save_append_using_path(self):
+        single_col_df = spark.createDataFrame([(4, ), (5, )])
+        # Fetch managed table path and attempt to side-step UC
+        # and directly update table using path based access.
+        tbl_path = spark.sql(
+            f"DESCRIBE formatted {CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}").collect()[5].data_type
+        try:
+            single_col_df.write.format("delta").save(mode="overwrite", path=tbl_path)
+        except Exception as error:
+            assert(S3_FORBIDDEN_ACCESS_ERROR in str(error))
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, self.setup_df)
 
-def create() -> None:
-    try:
-        spark.sql(f"CREATE TABLE {SCHEMA}.{TABLE_NAME} (a INT)")
-    except Exception:
-        print("[UNSUPPORTED] Creating managed table using UC commit coordinator isn't allowed")
+    # DataFrame V2 Tests #
+    def append(self, table_name):
+        single_col_df = spark.createDataFrame(
+            [(4, ),  (5, )], schema=StructType([StructField("id", IntegerType(), True)]))
+        single_col_df.writeTo(f"{CATALOG_NAME}.{SCHEMA}.{table_name}").append()
+        updated_tbl = read(table_name).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(1, ), (2, ), (3, ), (4, ), (5, )]))
```

Comment:
Name this `test_append` and create a dedicate helper method to append data that other tests can use